src/ui/layer_list_panel.css

```diff
@@ -33,6 +48,13 @@
   stroke: #999;
 }
 
+.neuroglancer-layer-side-panel-name {
```

Comment:
It would be better if we modify the CSS only for the name item when it is in the layer list panel. This is too broad and affects the name when it is in the layer side panel, and when it is part of a tool.

For example, we could add a class for this. Or update the selector